src/Castle.Core/Castle.Core-VS2017.csproj

```diff
@@ -10,6 +10,14 @@
     <AssemblyOriginatorKeyFile>..\..\buildscripts\CastleKey.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>True</SignAssembly>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <None Remove="DynamicProxy\DynProxy.snk" />
```

Comment:
Is the none remove actually needed, I'd have thought you'd only be able to have a file with one type or you'd have to make it `<Content Remove=...`.

If it has to stay probably best to put them in the same ItemGroup to make it obvious.